src/platforms/vue-native/scripts/compiler.js

```diff
@@ -248,3 +271,46 @@ function traverse(ast, nodes = []) {
     })
   }
 }
+
+function getFirstLineOfText(content, text) {
+  var firstLine = 0
+  let splittedContent = content.split(newLine)
+  if (!content.includes(text)) {
+    return splittedContent.length
+  }
+  splittedContent.some((line, index) => {
+    if (line.includes(text)) {
+      firstLine = index
+      return true
+    } else {
+      return false
+    }
+  })
+  return firstLine
+}
+
+function getFirstNonLineOfText(content, text) {
+  var firstNonCommentedLine = 0
+  let splittedContent = content.split(newLine)
+
+  if (!content.includes(text)) {
+    return 0
+  }
+  splittedContent.some((line, index) => {
```

Comment:
Code is confusing. `some` should be used to check for membership in an array.

Would be better to use [findIndex](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex)

Would also remove the need for the additional variable `firstNonCommentedLine`